DQMOffline/Trigger/python/TopMonitoring_cff.py

```diff
@@ -1,4 +1,149 @@
+eleHT_ele.nelectrons = cms.uint32(1)
+eleHT_ele.njets = cms.uint32(2)
+eleHT_ele.eleSelection = cms.string('pt>25 & abs(eta)<2.1')
+eleHT_ele.jetSelection = cms.string('pt>30 & abs(eta)<2.4')
+eleHT_ele.HTcut = cms.double(200)
+eleHT_ele.numGenericTriggerEventPSet.hltPaths = cms.vstring('HLT_Ele28_eta2p1_WPTight_Gsf_HT150_v*')
+eleHT_ele.denGenericTriggerEventPSet.hltPaths = cms.vstring('HLT_HT200_v*',
+                                                            'HLT_HT275_v*',)
+
+eleHT_all = hltTOPmonitoring.clone()
+eleHT_all.FolderName = cms.string('HLT/TopHLTOffline/TopMonitor/EleHT/GlobalMonitor')
+eleHT_all.nmuons = cms.uint32(0)
+eleHT_all.nelectrons = cms.uint32(1)
+eleHT_all.njets = cms.uint32(2)
+eleHT_all.eleSelection = cms.string('pt>25 & abs(eta)<2.1')
+eleHT_all.jetSelection = cms.string('pt>30 & abs(eta)<2.4')
+eleHT_all.numGenericTriggerEventPSet.hltPaths = cms.vstring('HLT_Ele28_eta2p1_WPTight_Gsf_HT150_v*')
+# eleHT_all.denGenericTriggerEventPSet.hltPaths = cms.vstring('HLT_IsoMu24_v*')
+
+
+#ATHER
+topSingleMuonHLTValidation = hltTOPmonitoring.clone()
+topSingleMuonHLTValidation.FolderName = cms.string('HLT/TopHLTOffline/TopMonitor/Top/SingleLepton/SingleMuon/')
+topSingleMuonHLTValidation.nmuons = cms.uint32(1)
+topSingleMuonHLTValidation.nelectrons = cms.uint32(0)
+topSingleMuonHLTValidation.njets = cms.uint32(4)
+topSingleMuonHLTValidation.eleSelection = cms.string('pt>30 & abs(eta)<2.5 & (dr03TkSumPt+dr03EcalRecHitSumEt+dr03HcalTowerSumEt)/pt < 0.1')
+topSingleMuonHLTValidation.muoSelection = cms.string('pt>26 & abs(eta)<2.1 & (pfIsolationR04.sumChargedHadronPt+pfIsolationR04.sumPhotonEt+pfIsolationR04.sumNeutralHadronEt)/pt < 0.12')
+topSingleMuonHLTValidation.jetSelection = cms.string('pt>20 & abs(eta)<2.5')
+topSingleMuonHLTValidation.numGenericTriggerEventPSet.hltPaths = cms.vstring(['HLT_Mu20_v*', 'HLT_TkMu20_v*' , 'HLT_Mu27_v*', 'HLT_TkMu27_v*', 'HLT_TkMu50_v*', 'HLT_Mu50_v*', 'HLT_IsoMu24_eta2p1_v*', 'HLT_IsoMu24_v*', 'HLT_IsoMu27_v*', 'HLT_IsoMu20_v*', 'HLT_IsoTkMu24_eta2p1_v*', 'HLT_IsoTkMu24_v*', 'HLT_IsoTkMu27_v*', 'HLT_IsoTkMu20_v*'])
+
+
+
+topDiElectronHLTValidation = hltTOPmonitoring.clone()
+topDiElectronHLTValidation.FolderName = cms.string('HLT/TopHLTOffline/TopMonitor/Top/DiLepton/DiElectron/')
+topDiElectronHLTValidation.nmuons = cms.uint32(0)
+topDiElectronHLTValidation.nelectrons = cms.uint32(2)
+topDiElectronHLTValidation.njets = cms.uint32(2)
+topDiElectronHLTValidation.eleSelection = cms.string('pt>20 & abs(eta)<2.5  & (dr03TkSumPt+dr03EcalRecHitSumEt+dr03HcalTowerSumEt)/pt < 0.1')
+topDiElectronHLTValidation.muoSelection = cms.string('pt>20 & abs(eta)<2.4 & (pfIsolationR04.sumChargedHadronPt+pfIsolationR04.sumPhotonEt+pfIsolationR04.sumNeutralHadronEt)/pt < 0.12')   
+topDiElectronHLTValidation.jetSelection = cms.string('pt>30 & abs(eta)<2.5')
+topDiElectronHLTValidation.numGenericTriggerEventPSet.hltPaths = cms.vstring(['HLT_Ele12_Ele23_CaloIdL_TrackIdL_IsoVL_DZ_v*'])
```

Comment:
sorry I actually replied to the wrong comment. I have to ask to the responsible for the di-lepton